src/js/worklet_factory.js

```diff
@@ -46,6 +46,8 @@ export function applyAnimationsToStyle(animations, style) {
         for (var transform of transforms) {
           var transformKey = Object.keys(transform)[0];
 	  var transformValue = transform[transformKey].value;
+      if (transformKey.startsWith('rotate'))
+        transformValue = `${transformValue}deg`
```

Comment:
Hi, Can we just pass rotate angle with 'deg' while using rotate transform?

Reply:
Hi @Parveshdhull,
It won't work if we do so. When we do something like `(str (reanimated/get-shared-value rotation) "deg")` it will be passed as a static value to the worklet. It will not animate.

Reply:
No, I mean just store orientation value with `deg` suffix in the shared value, for ex:
<div align="center">
<img src="https://user-images.githubusercontent.com/17097240/221488881-190870d4-15d2-4c54-8119-20b103dd2132.png" alt="Screenshot" height="400"/>
</div>

Reply:
Oh it can read handle strings. Perfect! Thank you